messaging/amqp-reactive/src/test/java/io/quarkus/ts/messaging/amqpreactive/BaseAmqpReactiveIT.java

```diff
@@ -21,6 +21,7 @@ public abstract class BaseAmqpReactiveIT {
      * Eventually, the consumer will get up to 10 prices (from 10 to 100) but it might receive more
      * {@link PriceConsumer#process(Integer price)}.
      */
+
```

Comment:
As you move the tests `ProdAmqpReactiveIT` can ve remove these additional blank lines in this files?